components/prism-dax.js

```diff
@@ -0,0 +1,27 @@
+Prism.languages.dax = {
+    'comment': {
+        pattern: /(^|[^\\])(?:\/\*[\s\S]*?\*\/|(?:--|\/\/).*)/,
+        lookbehind: true
+    },
+    'data-field': {
+        pattern: /'(?:[^']|'')*'(?!')(?:\[[ \w\xA0-\uFFFF]+\])?|\w+\[[ \w\xA0-\uFFFF]+\]/,
+        alias: 'symbol'
+    },
+    'measure': {
+        pattern: /[[ \w\xA0-\uFFFF]+\]/,
```

Comment:
Almost slipped by. Maybe some tests for `measure` as well?

```suggestion
        pattern: /\[[ \w\xA0-\uFFFF]+\]/,
```

Reply:
Indeed that was missing. Added as well.